packages/algob/src/lib/script-checkpoints.ts

```diff
@@ -209,13 +226,21 @@ export async function registerCheckpoints (
     switch (txn.type) {
       case 'acfg': {
         txConfirmation = await deployer.waitForConfirmation(txn.txID());
+        const key = deployer.getAssetCheckpointKeyFromIndex(txn.assetIndex);
+        if (key && checkAssetDeletionTx(txn)) {
+          const temp: rtypes.ASAInfo = deployer.getASAInfo(key);
+          temp.deleted = true;
+          deployer.registerASAInfo(key, temp);
+          break;
+        }
         if (res) {
           const asaInfo: rtypes.ASAInfo = {
             creator: encodeAddress(txn.from.publicKey),
             txId: txn.txID(),
             assetIndex: txConfirmation["asset-index"],
             confirmedRound: txConfirmation['confirmed-round'],
-            assetDef: res[1]
+            assetDef: res[1],
```

Comment:
Can we add a comment about `res[1]` ? That res[0] is name and res[1] is rtypes.ASADef